pkg/util/bs/bs.go

```diff
@@ -0,0 +1,19 @@
+package bs
+
+import (
+	"reflect"
+	"unsafe"
+)
+
+// String2Bytes converts string to byte slice without a memory allocation.
+func String2Bytes(s string) (b []byte) {
```

Comment:
Nit: for both of these methods, let's just use the word `To` instead of `2`. It is clearer and only a single character longer.

`StringToBytes` and `BytesToString`.

Reply:
bbe1d75